services/mempool/check.go

```diff
@@ -326,11 +326,6 @@ func checkInputsStandard(tx *types.Tx, utxoView *blockchain.UtxoViewpoint) error
 					i, numSigOps, maxStandardP2SHSigOps)
 				return txRuleError(message.RejectNonstandard, str)
 			}
-
-		case txscript.NonStandardTy:
```

Comment:
The safe guard check of `NoStandardTy` should not removed directly, we should add more specified tx type to allow. instead of allowing all the others case.

Reply:
https://github.com/Qitmeer/qitmeer/pull/481#issuecomment-824517038